features/find_doi_requests.feature

```diff
@@ -0,0 +1,53 @@
+Feature: Find DOI requests
+
+  Scenario Outline: A Creator finds DOI Requests
+    Given A Creator has Publications with DOI Requests
+    When they set the Accept header to "application/json"
+    And they set the Authentication header to a Bearer token with their credentials
+    And they request GET /doi/request?role=creator
+    Then they receive a response with status code 200
+    And they see that the response Content-Type header is "application/json"
+    And they see that the response body is a DoiRequestsResponse
+    And they see that the response body has a list of DOI Requests
+    And they see that each DOI Request has a property <Property>
+
+    Examples:
+      | Property              |
+      | doiRequestStatus      |
+      | doiRequestDate        |
+      | publicationIdentifier |
+      | publicationTitle      |
+      | publicationCreator    |
+
+  Scenario: A Curator finds DOI Requests
+    Given A Curator belongs to a Publisher that has Publications with DOI Requests
+    When they set the Accept header to "application/json"
+    And they set the Authentication header to a Bearer token with their credentials
+    And they request GET /doi/requests?role=curator
+    Then they receive a response with status code 200
+    And they see that the response Content-Type header is "application/json"
+    And they see that the response body is a DoiRequestsResponse
+    And they see that the response body has a list of DOI Requests
```

Comment:
er dette doi-requests for én publisher? burde ha mulighet til å hente alle doi-requests for alle publishers i sin "organisasjon"

Reply:
Eller er publisher selve "organisasjonen" her?

Reply:
publisher er publication.publisher, så det blir organisasjonen som eier publikasjonen.

Reply:
ok 👍